Gemfile.lock

```diff
@@ -353,7 +353,7 @@ DEPENDENCIES
   rails-html-sanitizer
   rails_12factor
   rainbow
-  react_on_rails (~> 0.1.8)
+  react_on_rails (~> 1.0.0.pre)
```

Comment:
There's something wrong with this PR, as this is already on master: 

https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/Gemfile.lock#L356

@rstudner Any ideas?